python/ray/tune/tests/test_searchers.py

```diff
@@ -232,22 +232,32 @@ def testNevergradWithRequiredOptimizerKwargs(self):
 
     def testOptuna(self):
         from optuna.samplers import RandomSampler
+        from optuna.storages import JournalStorage
+        from optuna.storages.journal import JournalFileBackend
 
         from ray.tune.search.optuna import OptunaSearch
 
         np.random.seed(1000)  # At least one nan, inf, -inf and float
+        storage_file_path = "/tmp/my_test_study.log"
 
         with self.check_searcher_checkpoint_errors_scope():
             out = tune.run(
                 _invalid_objective,
-                search_alg=OptunaSearch(sampler=RandomSampler(seed=1234)),
+                search_alg=OptunaSearch(
+                    sampler=RandomSampler(seed=1234),
+                    study_name="my_test_study",
+                    storage=JournalStorage(
+                        JournalFileBackend(file_path=storage_file_path)
+                    ),
+                ),
```

Comment:
One comment here is to test both cases when the `storage` arg is set or is None. We need to make sure the tests pass under both scenarios. Maybe consider using `@pytest.mark.parametrize` fixture to enable both cases. 

Here is one example of how to use the fixtures. https://github.com/ray-project/ray/blob/ed3d48c552e0ac2f235773322686c6ad5c521cd5/python/ray/tune/tests/test_trial.py#L25